homeworks/08-functional-reactive-programming/src/state/state.ts

```diff
@@ -0,0 +1,20 @@
+export interface IAction {
+    type: string,
+    payload?: any
+}
+
+enum ActionType {
+    Add = "Add",
+    Null = "Null"
+}
+
+export const Add  = () => ({ type: ActionType.Add  })
+export const Null = () => ({ type: ActionType.Null })
```

Comment:
Too common names. Might be replaced by `IncrementRoachCounter` & `ResetRoachCounter` or similar.